### ISSUE TITLE
Do not use green checkmark when checks fail.

### ISSUE THREAD

**Issue by vertti** [integrations/slack#1392]
When you have checks configure for your PR, Slack Github integration prints a message saying something like:
:white_check_mark: No checks have passed
0/4 successful checks

I think this is very misleading to look at a glance.

**Describe the solution you'd like**

If all checks pass, message should start with green check mark. If any checks fail, it should have for example 🔴  or 🟥  or ❌...

**Describe alternatives you've considered**

Maybe failing checks should get yellow X instead?

**Additional context**

In short, it is too easy to confuse this:
<img width="227" alt="Screenshot 2022-05-03 at 13 17 08" src="https://user-images.githubusercontent.com/557751/166440604-eab33106-c5e7-4cae-bb83-0af178edbd1f.png">

and this:

<img width="228" alt="Screenshot 2022-05-03 at 13 17 01" src="https://user-images.githubusercontent.com/557751/166440621-576cf341-7910-4a84-b5a8-63f487591d9b.png">
